MediaBrowser.Model/System/SystemStorageInfo.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Model.System;
+
+/// <summary>
+/// Contains informations about the systems storage.
+/// </summary>
+public class SystemStorageInfo
+{
+    /// <summary>
+    /// Gets or sets the program data path.
```

Comment:
All these summaries _could_ do with some better descriptions. Some are obvious what they do but I have no clue what "ItemsByName" does for example. The descriptions are directly copied over to the OpenAPI specification and thus the ApiClients so they **are** the API documentation.